bodhi/tests/server/test_mail.py

```diff
@@ -287,7 +287,7 @@ def test_recipients_refused(self, SMTP, warn):
         SMTP.assert_called_once_with('smtp.fp.o')
         smtp.sendmail.assert_called_once_with('archer@spies.com', ['lana@spies.com'], 'hi')
         warn.assert_called_once_with(
-            '"recipient refused" for \'lana@spies.com\', SMTPRecipientsRefused(\'nooope!\',)')
+            '"recipient refused" for \'lana@spies.com\', SMTPRecipientsRefused(\'nooope!\'{})'.format(base._erepr_end))
```

Comment:
'"recipient refused" for \'lana@spies.com\', {}'.format(smtplib.SMTPRecipientsRefused('nooope!'))
?